model/project_ref.go

```diff
@@ -3215,6 +3215,37 @@ func ValidateContainerSecrets(settings *evergreen.Settings, projectID string, or
 	return combined, catcher.Resolve()
 }
 
+// ValidateParsleyFilters checks that there are no duplicate expressions among the Parsley filters. It also validates
+// each individual Parsley filter.
+func ValidateParsleyFilters(parsleyFilters []ParsleyFilter) error {
+	catcher := grip.NewBasicCatcher()
+
+	filtersSet := make(map[string]bool)
+	for _, filter := range parsleyFilters {
+		if filtersSet[filter.Expression] {
+			catcher.Errorf("duplicate filter expression '%s'", filter.Expression)
+			break
```

Comment:
Can we continue rather than break in order to get all of the duplicates instead of exiting at the first duplicate we find?